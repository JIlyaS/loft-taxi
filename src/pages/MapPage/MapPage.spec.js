// import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import {MapPage} from './';

describe('MapPage', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<MapPage page="map" navigateTo={() => {}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // не проходит из-за this.map.remove()
  // it('render correct', () => {
  //   const { queryByTestId } = render(<MapPage page="map" navigateTo={() => {}} />);
  //   expect(queryByTestId('main')).toBeInTheDocument();
  // });
});
