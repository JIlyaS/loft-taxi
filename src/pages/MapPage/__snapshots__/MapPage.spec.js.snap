// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapPage renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <img
      alt="Логотип"
      className="header__logo"
      src="logo.svg"
    />
    <nav
      className="header__nav"
    >
      <ul
        className="header__nav-list"
      >
        <li
          className="header__nav-item"
        >
          <a
            className="header__nav-link header__nav-link--active"
            onClick={[Function]}
          >
            Карта
          </a>
        </li>
        <li
          className="header__nav-item"
        >
          <a
            className="header__nav-link"
            onClick={[Function]}
          >
            Профиль
          </a>
        </li>
        <li
          className="header__nav-item"
        >
          <a
            className="header__nav-link"
            onClick={[Function]}
          >
            Выйти
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <main
    className="main-page"
  >
    <div
      className="map"
    >
      <div
        className="map__component"
        data-testid="map"
      />
    </div>
  </main>,
]
`;
